src/SourceBuild/tarball/content/repos/test-templates.proj

```diff
@@ -1,19 +1,13 @@
 <Project>
-
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <BuildCommandArgs>--restore --build --pack</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) --configuration $(Configuration)</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) --binaryLog</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) -ci</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) $(FlagParameterPrefix)nodereuse $(ArcadeFalseBoolBuildArg)</BuildCommandArgs>
-    <BuildCommand>$(ProjectDirectory)build$(ShellExtension) $(BuildCommandArgs)</BuildCommand>
+    <BuildCommandArgs>$(StandardSourceBuildArgs) $(FlagParameterPrefix)nodereuse $(ArcadeFalseBoolBuildArg)</BuildCommandArgs>
+    <BuildCommand>$(StandardSourceBuildCommand) $(BuildCommandArgs)</BuildCommand>
 
     <GlobalJsonFile>$(ProjectDirectory)global.json</GlobalJsonFile>
     <NuGetConfigFile>$(ProjectDirectory)NuGet.config</NuGetConfigFile>
     <LogVerbosityOptOut>true</LogVerbosityOptOut>
-    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
```

Comment:
I'm a little concerned about removing this.  It looks to me like this is defaulted to true, so removing this will cause the `CopyPackage` target to not be executed and the output nupkgs won't be copied.

Reply:
You are correct, I was under the impression that `DotNetOutputBlobFeedDir` was handling this when it doesn't.  `<OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>` should be the default/only behavior with ArPow now.  I will made note of this in https://github.com/dotnet/source-build/issues/2306.